planning/behavior_path_planner/src/utils/utils.cpp

```diff
@@ -621,6 +621,29 @@ bool checkCollisionBetweenPathFootprintsAndObjects(
   return false;
 }
 
+bool checkCollisionWithExtraStoppingMargin(
```

Comment:
@kosuke55
could you move this util function in safety_check.cpp?

Reply:
@kyoichi-sugahara OK! that's better!

Reply:
moved in https://github.com/autowarefoundation/autoware.universe/pull/4674/commits/accb2dfd9bfd1c4be772e6b069e5e9a970ee4660